python/py-jsonschema_specifications/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-jsonschema_specifications
+version             2023.7.1
+revision            0
+categories-append   devel
+license             MIT
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     38 39 310 311
+python.pep517       yes
+python.pep517_backend hatch
+
+maintainers         openmaintainer
+
+description         "JSON support files from the JSON Schema Specifications (metaschemas, vocabularies, etc.),
+packaged for runtime access from Python as a referencing-based Schema Registry."
+long_description    {*}${description}
+
+homepage            https://github.com/python-jsonschema/jsonschema-specifications
+
+checksums           rmd160  c3533cbd28ecf7856222ec13b3c5ebe8039cf3ce \
+                    sha256  c91a50404e88a1f6ba40636778e2ee08f6e24c5613fe4c53ac24578a5a7f72bb \
+                    size    12689
+
+if {${name} ne ${subport}} {
+    depends_build-append  port:py${python.version}-hatch-vcs
+    depends_lib-append    port:py${python.version}-referencing
+
+    if {${python.version} < 39} {
+        depends_lib-append  port:py${python.version}-importlib-resources
+    }
```

Comment:
please verify that everything is aligned with "four spaces"; it doesn't look like the dependencies here are